openslides/users/apps.py

```diff
@@ -29,5 +29,5 @@ def ready(self):
         register_slide_model(User, 'participant/user_slide.html')
 
         # Register viewsets.
-        router.register('users/user', UserViewSet)
-        router.register('users/group', GroupViewSet)
+        router.register('user/user', UserViewSet)
```

Comment:
Hm. The app name is 'users' not 'user'. We also wanted to rename all other apps to plural names (motions, assignments, ...) as discussed in October 2014.